src/themes/dojo/comboBox.m.css

```diff
@@ -1,55 +1,103 @@
 @import './variables.css';
 
 .root {
-	position: relative;
-	display: inline-block;
 	box-sizing: border-box;
-	width: var(--form-field-width);
+	color: var(--color-text-primary);
+	display: inline-block;
+	font-size: var(--font-size-base);
+	line-height: var(--line-height-base);
+	position: relative;
+	width: 100%;
 }
 
 .root * {
 	box-sizing: border-box;
 }
 
-.controls {
-	position: relative;
+.root input {
+	line-height: var(--line-height-base);
+	min-width: 200px;
+	padding: var(--spacing-regular) calc(var(--spacing-regular) * 6) var(--spacing-regular) var(--spacing-regular);
+	width: 100%;
+}
+
+.trigger {
```

Comment:
Totally agreed. We have dojo/widgets#117 to track that progress, as it's more involved than just pulling out common styling. The underlying DOM is still somewhat different and would require some consolidation.

I definitely think we need to take care of that ticket asap though.